src/format_test.cc

```diff
@@ -77,4 +93,57 @@ TEST_F(FormatTest, RowCount) {
   EXPECT_EQ(3U, fmt->RowCount());
 }
 
+struct StdData {
+  const char* name;
+  const char* fmt;
+  uint32_t column_count;
+  bool is_std140;
+  uint32_t values_per_element;
+  uint32_t size_in_bytes_per_row;
+  uint32_t size_in_bytes;
```

Comment:
I think "size_in_bytes == values_per_element * sizeOf(float)"
And "size_in_bytes_per_row = size_in_bytes / column_count".
In that case could we calculate them instead of writing the values?

Reply:
My preference for tests is to be explicit and wordy. So, this looks terrible but is very explicit on what it's expecting.